libr/arch/p/m680x_cs/plugin.c

```diff
@@ -67,6 +67,21 @@ static inline csh cs_handle_for_session (RArchSession *as) {
 	return pd->cs_handle;
 }
 
+static inline char *filter_intel_syntax (char *mnemonic) {
```

Comment:
i think its better not to inline this
```suggestion
static char *filter_intel_syntax(char *mnemonic) {
```

Reply:
ok!